trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -305,42 +460,155 @@ def VLM(conditions,settings,geometry):
-    CL       = np.atleast_2d(np.sum(LIFT,axis=1)/SREF).T
-    CDi      = np.atleast_2d(np.sum(DRAG,axis=1)/SREF).T
-    CM       = np.atleast_2d(np.sum(MOMENT,axis=1)/SREF).T/c_bar
-
     alpha_i = np.hsplit(np.arctan(cdi_y/cl_y),span_breaks[1:])
     
-    return CL, CDi, CM, CL_wing, CDi_wing, cl_y, cdi_y, alpha_i, CP, gamma
\ No newline at end of file
+    # Now calculate total coefficients
+    CL       = np.atleast_2d(np.sum(LIFT,axis=1)/SREF).T          # CLTOT in VORLAX
+    CDi      = np.atleast_2d(np.sum(DRAG,axis=1)/SREF).T          # CDTOT in VORLAX
+    CM       = np.atleast_2d(np.sum(MOMENT,axis=1)/SREF).T/c_bar  # CMTOT in VORLAX
+
+    CYTOT    = np.atleast_2d(np.sum(FY,axis=1)/SREF).T   # total y force coeff
+    CRTOT    = np.atleast_2d(np.sum(RM,axis=1)/SREF).T   # total rolling moment coeff
+    CRMTOT   = CRTOT/w_span*(-1)                         # an output in VORLAG.LOG
+    CNTOT    = np.atleast_2d(np.sum(YM,axis=1)/SREF).T   # total yawing  moment coeff
+    CYMTOT   = CNTOT/w_span*(-1)                         # an output in VORLAG.LOG
+
+    # ---------------------------------------------------------------------------------------
+    # STEP 13: Pack outputs
+    # ------------------ --------------------------------------------------------------------     
+    precision      = settings.floating_point_precision
+    
+    #VORLAX _TOT outputs
+    results = Data()
+    results.CL         =  CL         
+    results.CDi        =  CDi        
+    results.CM         =  CM  
+    results.CYTOT      =  CYTOT
+    results.CRTOT      =  CRTOT
+    results.CRMTOT     =  CRMTOT
+    results.CNTOT      =  CNTOT
+    results.CYMTOT     =  CYMTOT
+    
+    #other SUAVE outputs
+    results.CL_wing    =  CL_wing   
+    results.CDi_wing   =  CDi_wing 
+    results.cl_y       =  cl_y     
+    results.cdi_y      =  cdi_y     
+    results.alpha_i    =  alpha_i  
+    results.CP         =  np.array(CP    , dtype=precision)
+    results.gamma      =  np.array(GAMMA , dtype=precision)
+    
+    #append inputs
+    results.conditions = conditions
+    results.settings   = settings
+    results.geometry   = geometry
+    
+    return results
+
+# ----------------------------------------------------------------------
+#  CLE rotation effects helper function
+# ----------------------------------------------------------------------
+def compute_rotation_effects(VD, settings, EW_small, GAMMA, len_mach, X, CHORD, XLE, XBAR, 
+                             rhs, COSINP, SINALF, PITCH, ROLL, YAW, STB, RNMAX):
+    """ This computes the effects of the freestream and aircraft rotation rate on 
+    CLE, the induced flow at the leading edge
+    
+    Assumptions:
+    Several of the values needed in this calculation have been computed earlier in
+    either compute_RHS_matrix() or generate_vortex_distribution() and stored in VD
+    """
+    spacing     = settings.spanwise_cosine_spacing
```

Comment:
variable is not used

Reply:
ah yeah, I'll put that one inside the TODO comment right underneath it